DEPS.md

```diff
@@ -0,0 +1,64 @@
+## Dependency Management
+
+Vendoring all dependencied is essential to have a **go get**-able package.
+
+Tools needed:
+
+- [glide](github.com/Masterminds/glide) dependency manager
+- [glide-vc](github.com/sgotti/glide-vc) vendor cleaner plugin for glide
+
+`make deps` will instal them, in case they are missing
+
+## Add a new dependency
+
+If you write new features which imports a new library, you have to vendor it:
+```
+glide get -v github.com/Masterminds/cookoo/web
+glide-vc --only-code
```

Comment:
Shall we not add `--no-tests` as well to remove tests from vendor?

Reply:
good catch